src/ncp/ncp_base_dispatcher.cpp

```diff
@@ -411,6 +411,12 @@ NcpBase::PropertyHandler NcpBase::FindSetPropertyHandler(spinel_prop_key_t aKey)
 #if OPENTHREAD_CONFIG_MAC_RETRY_SUCCESS_HISTOGRAM_ENABLE
         OT_NCP_SET_HANDLER_ENTRY(SPINEL_PROP_CNTR_MAC_RETRY_HISTOGRAM),
 #endif
+#endif // OPENTHREAD_MTD || OPENTHREAD_FTD
+#if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
+        OT_NCP_SET_HANDLER_ENTRY(SPINEL_PROP_RCP_MAC_KEY),
+        OT_NCP_SET_HANDLER_ENTRY(SPINEL_PROP_RCP_MAC_KEY_ID),
+#endif
```

Comment:
What about putting these lines after line 428? Thus we can separate the `OPENTHREAD_FTD || OPENTHREAD_MTD` part from the `OPENTHREAD_RADIO` part.

Reply:
I think there is a check to see if these properties are in order with their value. Moving these around will throw an expect error.

Reply:
Yes, this list needs be sorted since binary search is used to find the matching handler for a property key.